sql_builder.go

```diff
@@ -114,6 +121,27 @@ func (sq *sqlBuilder) buildSelectSQL() string {
 	return sql
 }
 
+func (sq *sqlBuilder) buildDeleteSQL() string {
+	fc := sq.buildfromClauses()
+
+	sql := fmt.Sprintf("DELETE FROM %s", fc)
+
+	sql = sq.buildWhereClauses(sql)
+
+	// paginated delete supported by sqlite and mysql
+	// > If SQLite is compiled with the SQLITE_ENABLE_UPDATE_DELETE_LIMIT compile-time option [...] - from https://www.sqlite.org/lang_delete.html
+	//
+	// not generic enough IMO, therefore excluded
+	//
+	//switch sq.Query.Connection.Dialect.Name() {
+	//case nameMySQL, nameSQLite3:
+	//	sql = sq.buildOrderClauses(sql)
+	//	sql = sq.buildPaginationClauses(sql)
+	//}
```

Comment:
Not sure about this part, but I think it is better to leave it out. Should I delete the comment?

Reply:
Yeah leave the feature out, but the comment in :)